# HG changeset patch
# User Roman Kraevskiy <rkraevskiy@gmail.com>
# Date 1237455364 -3600
# Branch HEAD
# Node ID 6fac57b97bf1e829d1bfdf643583f5921e153fea
# Parent  17d770ae41e4bc94527181dc62072564b9fbd252
Restore connection polling callback when closing SASL connection. Closes #3206.

diff -r 17d770ae41e4 -r 6fac57b97bf1 mutt_sasl.c
--- a/mutt_sasl.c	Wed Mar 18 14:48:07 2009 +0100
+++ b/mutt_sasl.c	Thu Mar 19 10:36:04 2009 +0100
@@ -481,6 +481,7 @@
   conn->conn_close = sasldata->msasl_close;
   conn->conn_read = sasldata->msasl_read;
   conn->conn_write = sasldata->msasl_write;
+  conn->conn_poll = sasldata->msasl_poll;
 
   /* release sasl resources */
   sasl_dispose (&sasldata->saslconn);
